### PR TITLE
SWAP-1244 Github actions: Include cypress screenshots as build artifacts for failing e2e tests

### PR DESCRIPTION
**Pull request by peter-asztalos**
## Description

This PR adds a new step in Github actions, if e2e testing fails it will upload the screenshots taken by cypress as an artifact for each individual run.

## Motivation and Context

Help debug falling e2e tests.

## How Has This Been Tested

- [x] Manual testing

An example can be found here: https://github.com/UserOfficeProject/user-office-scheduler-frontend/actions/runs/385111575

## Fixes

https://jira.esss.lu.se/browse/SWAP-1244

## Changes

- new step added to Github actions

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: .github/workflows/test-build.yml [hunk 1]
```diff
@@ -173,6 +173,14 @@ jobs:
             -f user-office-gateway/docker-compose.e2e.yml \
             -f "$REPO_DIR_NAME/docker-compose.e2e.yml" \
             up --exit-code-from cypress
+
```
Comment (jekabs-karklins) [inline .github/workflows/test-build.yml:176]: How can we access the screenshots afterwards?

Comment (peter-asztalos) [reply .github/workflows/test-build.yml:176]: If you click on the action name above the action steps (`Build, Test and Push` in this case) you will be redirected to the selected Github action run page, where is an `Artifacts` panel, under that should be a `cypress-screenshots` file.
You can access from PR Actions tab or by selecting a run under the repo's Action tab.

Comment (peter-asztalos) [reply .github/workflows/test-build.yml:176]: I included an example link: https://github.com/UserOfficeProject/user-office-scheduler-frontend/actions/runs/385111575

Comment (jekabs-karklins) [reply .github/workflows/test-build.yml:176]: Nice

**Review by jekabs-karklins — APPROVED** [review]

**Review by martin-trajanovski — APPROVED** [review]

**Pull request merged**